### ISSUE TITLE
CORS warning icon not rendered well

### ISSUE THREAD

**Issue by chrisdudley** [swagger-api/swagger-editor#538]
The CORS warning message includes a warning icon.
That icon uses an embedded unicode character that certainly on some machines I've tried doesn't render and I just see an empty rectangle.

Maybe switch to the HTML entity instead? 
&#9888;

**Comment by chrisdudley** [comment]
Doh - eaten by the parser... let me try again:

&amp;#9888;

See http://www.fileformat.info/info/unicode/char/26a0/index.htm

**Comment by mohsen1** [comment]
If your browser can't render the unicode character, it shouldn't be any different if you use HTML code point syntax. Does changing  `⚠` to `&#9888;` fix the issue for you?

**Comment by chrisdudley** [comment]
You're right. It doesn't. Had assumed that would fix it.
Must be another chrome / unicode incompatibility issue. Works fine in FF.

**Comment by mohsen1** [comment]
We opted to use Unicode characters for icons for simplicity. Some browsers might not render them perfectly but I'm sure all browsers will get there soon. Since it's not critical to editor's functionality we can leave it as is. 
There is also [`charset`](https://github.com/swagger-api/swagger-editor/blob/master/app/index.html#L3) meta tag that tells browser to use UTF-8.

**Comment by chrisdudley** [comment]
Aha the solution was to force the font-family to one that includes that icon and so works in chrome.
In this case:
font-family: DejaVu Sans, Symbola, Everson Mono, Dingbats, Segoe UI Symbol,
 Quivira, SunExt-A, FreeSerif, Universalia, unifont;
